app/interfaces/api/entities/offence.rb

```diff
@@ -9,8 +9,13 @@ class Offence < BaseEntity
       expose :full_format, merge: true, unless: export_format? do
         expose :id
         expose :description
-        expose :offence_class_id
-        expose :offence_class, using: API::Entities::OffenceClass
+        expose :offence_class_id, if: ->(instance, _opts) { instance.fee_schemes.first.number.eql?(9) }
+        expose :offence_class,
+               if: ->(instance, _opts) { instance.fee_schemes.first.number.eql?(9) },
```

Comment:
[Ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2283/files#r175754917)

Reply:
**[not a blocker]** Also, 💭  consider moving the responsibility of checking the fee scheme to the offence itself:

```ruby
# made up name, feel free to come up with a better one :P
if: ->(instance, _opts) { instance.on_scheme?(FeeScheme::AGFS_SCHEME9) },
```